### ISSUE TITLE
Blog about Horizon (and open-source the codebase)

### ISSUE THREAD

**Issue by cvan** [MozVR/mozvr-site#150]
Our WebVR-first, backwards-compatible browser project called ["Horizon"](https://github.com/mozvr/horizon) ([video](https://vimeo.com/157952401)) 

And there's definitely more related topics of previous projects to feature in some blog posts at some point too. Josh has a good recap on his site: http://www.joshcarpenter.ca/vr-browsing-explorations/

**Comment by caseyyee** [comment]
Good idea.  We have a bunch of experiments and code in places that has been lost and forgotten about that can be used.   Along the same lines:  https://www.youtube.com/watch?v=7-72zb81QFw
